frontend/src/scenes/trends/viz/ActionsBarValueGraph.tsx

```diff
@@ -65,6 +67,18 @@ export function ActionsBarValueGraph({
                 labels={data[0].labels}
                 dashboardItemId={dashboardItemId}
                 totalValue={total}
+                onClick={
+                    dashboardItemId
+                        ? null
+                        : (point) => {
+                              const { dataset } = point
+                              const action = dataset.actions[point.index]
+                              const label = dataset.labels[point.index]
+                              const date_from = dataset.days[0]
+                              const date_to = dataset.days[dataset.days.length - 1]
+                              loadPeople(action || 'session', label, date_from, date_to, null)
```

Comment:
I realized that the payload we return for bar charts by value simply uses an aggregated result so the `dataset.days` array is actually empty because we don't need time series context as we normally would in the other payloads. 

A few ways you can proceed:
1. Propagate those days in the payload from the backend like we do for regular trend graphs
2. Parse the date interval set in the filters on the frontend and just insert that here/in the logic when the call is being made